vue-components/src/components/Checkbox.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+	<span class="wikit wikit-checkbox">
+		<input
+			type="checkbox"
+			:id="id"
+			class="wikit-checkbox__input"
+			@change="$emit( 'input', $event.target.checked )"
+			:value="nativeValue"
+			:checked="value"
```

Comment:
I’m very much against switching the native attribute names again in this component (as it happened in Button).
`checked` can just be called `checked` or `isChecked` if you wish and `value` can be `value` instead of `nativeValue`.
Also, I can't quite explain it, but having a boolean variable named `value` seems strange to me.

Reply:
The motivation for connecting the `value` prop with the `checked` attribute was to enable the application to use this component directly with `v-model`.

Though, if we were to drop that concern, then I agree that we should choose the props as you suggest.

Reply:
@micgro can you exemplify this concern? How is changing the names back going to affect it?

Reply:
With how it is now:

```vue
<template>
	<Checkbox v-model="isChecked" label="loremIpsum">
</template>

<script>
export default Vue.extend( {
	name: 'QueryBuilder',
	data() {
		return {
			isChecked: false,
		};
	},
} );
</script>
```

With the props changed like suggested by Tonina:
```vue
<template>
	<Checkbox :isChecked="isChecked" @input="isChecked = event" label="loremIpsum">
</template>

<script>
export default Vue.extend( {
	name: 'QueryBuilder',
	data() {
		return {
			isChecked: false,
		};
	},
} );
</script>
```

I'm ok with the second, I just want us to make that trade-off consciously.

Also, this might no longer be a problem in vue3? I think v-model changed there to be more flexible, but I'm not sure.

Reply:
Honestly I am also okay with both, I slightly like the first one better due to brevity, but since @tozh raised this issue I'll wait to see what she thinks.

Reply:
I think this won't be a problem in vue3, because `v-model` [can accept arguments](https://v3.vuejs.org/guide/migration/v-model.html#v-model-arguments).
On the other hand, I have no idea if there are plans  to migrate the vue components to vue3 anytime soon, so I don't know if that should be taken into consideration yet. 
IMHO the name of the props take precedent over the shorthand syntax when using the Checkbox component.
If others think differently then please go ahead with the current version, I don't want to block your work over something like this. :v:

Reply:
I have to touch this PR again anyway to add the new Label mixin, so I can update the props here as well. I will update the emitted event to be `updated:checked`, so that the final usage will be:

vue2:
```vue
<template>
	<Checkbox :checked.sync="isChecked" label="loremIpsum">
</template>

<script>
export default Vue.extend( {
	name: 'QueryBuilder',
	data() {
		return {
			isChecked: false,
		};
	},
} );
</script>
```

vue3:
```vue
<template>
	<Checkbox v-model:checked="isChecked" label="loremIpsum">
</template>

<script>
export default Vue.extend( {
	name: 'QueryBuilder',
	data() {
		return {
			isChecked: false,
		};
	},
} );
</script>
```

I think that will give us the best of both worlds.